app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -176,25 +176,29 @@ def parse_openshift_image(openshift_image)
         ref = ContainerImage::DOCKER_PULLABLE_PREFIX + id
```

Comment:
So if id ends up being `""`, ref ends up being `"docker-pullable://"`,
does `parser_container_image` which calls `parse_image_name` all work?

Can you add some unit test where at least some of these things are missing?